### PR TITLE
feat: Store message on sequencer DB

### PR DESCRIPTION
**Pull request by ChaituVR**
Related to https://www.notion.so/snapshotlabs/Move-all-messages-to-sequencer-f7843f0b1f3049b79f87647ca652d181

### Changes: 
- Save message to hub DB and sequencer DB 

### Important
Add following `env` to digitalocean
- [x] Rename `DATABASE_URL` to `HUB_DATABASE_URL`
- [x] Add new variable `SEQ_DATABASE_URL`

### CODE REVIEW

File: src/helpers/mysql.ts [hunk 1]
```diff
@@ -9,17 +9,32 @@ const connectionLimit = parseInt(process.env.CONNECTION_LIMIT ?? '25');
...
+sequencerConfig.port = sequencerConfig.hosts[0].port;
+sequencerConfig.connectTimeout = 60e3;
+sequencerConfig.acquireTimeout = 60e3;
+sequencerConfig.timeout = 60e3;
+sequencerConfig.charset = 'utf8mb4';
+bluebird.promisifyAll([Pool, Connection]);
+const sequencerDB = mysql.createPool(sequencerConfig);
+
+export default hubDB;
+export { hubDB, sequencerDB };
```
Comment (wa0x6e) [inline src/helpers/mysql.ts:40]:
```suggestion
export { hubDB as default, sequencerDB };
```

File: src/helpers/highlight.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import db from './mysql';
+import { hubDB, sequencerDB } from './mysql';
```
Comment (wa0x6e) [inline src/helpers/highlight.ts:1]:
```suggestion
import { default as hubDB, sequencerDB } from './mysql';
```

**Review by wa0x6e — CHANGES_REQUESTED** [review]

File: src/helpers/mysql.ts [hunk 3]
```diff
@@ -9,17 +9,31 @@ const connectionLimit = parseInt(process.env.CONNECTION_LIMIT ?? '25');
...
+sequencerConfig.multipleStatements = true;
+sequencerConfig.database = sequencerConfig.path[0];
+sequencerConfig.host = sequencerConfig.hosts[0].name;
+sequencerConfig.port = sequencerConfig.hosts[0].port;
+sequencerConfig.connectTimeout = 60e3;
+sequencerConfig.acquireTimeout = 60e3;
+sequencerConfig.timeout = 60e3;
+sequencerConfig.charset = 'utf8mb4';
+bluebird.promisifyAll([Pool, Connection]);
+const sequencerDB = mysql.createPool(sequencerConfig);
```
[the remark this answers is not in the document]
Comment (wa0x6e) [reply src/helpers/mysql.ts:37]: should `bluebird.promisifyAll([Pool, Connection]);` run multiple time ?

File: src/helpers/highlight.ts [hunk 4]
```diff
@@ -15,4 +15,20 @@ export async function storeMsg(id, ipfs, address, version, timestamp, space, typ
...
+        address,
+        version,
+        timestamp,
+        space,
+        type,
+        sig,
+        receipt
+      }
+    ]);
+  }
```
Comment (wa0x6e) [inline src/helpers/highlight.ts:33]: I assume this PR is to make the messages save in both DB at the same time, and another PR will remove the old insert into the hub db ?

File: src/helpers/mysql.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (Todmy) [reply src/helpers/mysql.ts:37]: true. Missed that. Can be moved outside the function

File: src/helpers/highlight.ts (hunk 4 above)
Comment (ChaituVR) [reply src/helpers/highlight.ts:33]: Correct, to make sure there is 0 downtime.

File: src/helpers/mysql.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (Todmy) [reply src/helpers/mysql.ts:37]: I think it's can be skipped since the solution is temporary

**Review by Todmy — APPROVED** [review]
utACK

**Review by wa0x6e — APPROVED** [review]

**Pull request merged**